hack/common.sh

```diff
@@ -64,7 +64,6 @@ UPGRADE_FROM_OCS_REGISTRY_IMAGE="${UPGRADE_FROM_OCS_REGISTRY_IMAGE:-quay.io/ocs-
 UPGRADE_FROM_OCS_SUBSCRIPTION_CHANNEL="${UPGRADE_FROM_OCS_SUBSCRIPTION_CHANNEL:-$OCS_SUBSCRIPTION_CHANNEL}"
 
 OCS_MUST_GATHER_DIR="${OCS_MUST_GATHER_DIR:-ocs-must-gather}"
-OCP_MUST_GATHER_DIR="${OCP_MUST_GATHER_DIR:-ocp-must-gather}"
```

Comment:
> Also remove the collection of ocp must-gather as all the relevant information is already inside the ocs must-gather.
- unless it's adding considerable delay I'd suggest not to remove as not all info about a cluster is collected in ocs mg alone, wdyt?

Reply:
Forgot to add in the commit message, ocp-must-gather is always collected & stored in the artifacts dir by the post steps. I was referring to us collecting the ocp must gather again upon our test failure.

Reply:
Like here https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/red-hat-storage_ocs-operator/3431/pull-ci-red-hat-storage-ocs-operator-main-ocs-operator-bundle-e2e-aws/1954978141105557504/artifacts/ocs-operator-bundle-e2e-aws/gather-must-gather/artifacts/